server/selection_algorithm.go

```diff
@@ -49,7 +49,7 @@ func (sa ProbabilitySelectionAlgorithm) filterByPerfScore(addrs []ethcommon.Addr
 	if len(res) == 0 {
 		// If no orchestrators pass the perf filter, return all Orchestrators.
 		// That may mean some issues with the PerfScore service.
-		glog.Warning("No Orchestrators passed min performance score filter, not using the filter")
+		glog.Warning("No Orchestrators passed min performance score filter, not using the filter, minPerfScore=%v, scores=%v", sa.MinPerfScore, scores)
```

Comment:
WDYT of adding `addrs` or probably just `len(addrs)` to these logs? Should help debug the filtering pipeline. Or even we could always have this log, including `len(addrs)` and `len(res)`? (might wanna omit the `scores` tho as that's big)

Reply:
Yes, good idea, I'll add both `len(addrs)` and `addrs`. This should help us investigate the issue.

We don't need `len(res)`, it's always `0` at this point.